published-namespaces.txt

```diff
@@ -339,7 +346,7 @@ AndroidX.WebKit.ChromiumLibBoundary.Util
 AndroidX.WebKit.Internal
 AndroidX.Window
 AndroidX.Window.Area
-AndroidX.Window.Area.ReflectionGuard
+AndroidX.Window.Area.Reflectionguard
```

Comment:
`ReflectionGuard`

Reply:
I don't know how I missed that one.

Fixed